CMakeLists.txt

```diff
@@ -29,17 +29,22 @@ option(TREE_BUILD_TESTS "Build unit tests?" OFF)
 #option(WITH_TIFF "With libgeotiff support?" OFF)
 #ras_bool_to_int(WITH_TIFF)
 
+if(WITH_TIFF)

```

Comment:
I think you need to uncomment the 2 lines above (#option(WITH_TIFF....) so it exists as a variable